lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -29,11 +29,16 @@ export default function DashboardActivityFilters({
   const { dashboard } = useConfig(['dashboard']);
   const { routes } = dashboard;
 
-  const courses = useAPIFetch<{ courses: Course[] }>(routes.courses);
+  const courses = useAPIFetch<{ courses: Course[] }>(routes.courses, {
+    public_id: dashboard.organization_public_id || [],
```

Comment:
I was looking at the type definition in `useAPIFetch`, and I think it should allow keys with value `undefined`, which should be just ignored.

Right now it only allows `string` or `string[]`, so your `|| []` workaround should work.